Assets/MixedRealityToolkit/_Core/Definitions/Physics/RayStep.cs

```diff
@@ -43,6 +47,11 @@ public void CopyRay(Ray ray, float rayLength)
             Terminus = Origin + (Direction * Length);
         }
 
+        public bool Contains(Vector3 point)
+        {
+            return Vector3.Distance(Origin, point) + Vector3.Distance(point, Terminus) - Length < epsilon;
```

Comment:
pretty sure there's a built in value for this we can use instead?

Reply:
Yeah, I tried to use Unity's built in epsilon, but I was hitting slightly larger values than that supports.